src/components/Separator/index.tsx

```diff
@@ -0,0 +1 @@
+export { Separator as default } from './Separator';
```

Comment:
add 
`export type { SeparatorProps } from './Separator.types'`

Reply:
do we need to export it?